modules/priors/R/plots.R

```diff
@@ -191,38 +192,39 @@ plot.trait <- function(trait,
           axis.ticks.y = element_blank(), 
           axis.line.y = element_blank(), 
           legend.position = "none", 
-          plot.title = element_text(size = fontsize$title), 
+          plot_title = element_text(size = fontsize$title), 
```

Comment:
This one's a ggplot theme object and should stay dotted

Reply:
```suggestion
          plot.title = element_text(size = fontsize$title), 
```